src/posit/connect/client.py

```diff
@@ -1,15 +1,24 @@
 from __future__ import annotations
 
+import os
 from requests import Response, Session
 from typing import Optional
 
 from . import hooks, urls
 
 from .auth import Auth
 from .config import Config
+from .oauth import OAuthIntegration
 from .users import Users
 
 
+# Connect sets the value of the environment variable RSTUDIO_PRODUCT = CONNECT
```

Comment:
😭  it's 2024, we should start setting `POSIT_PRODUCT` in the env too, yeah?

Reply:
Can we use the presence of `CONNECT_SERVER` instead and sidestep branding issues?